sql/src/main/java/io/crate/planner/node/ExecutionPhase.java

```diff
@@ -70,6 +70,10 @@ public ExecutionPhase fromStream(StreamInput in) throws IOException {
 
     Type type();
 
+    default String typeStr() {
```

Comment:
Would it make sense to extend / use `ExplainNode` / `ExplainLeaf` instead and have `ExecutionPhase` extend one of those interfaces?

Reply:
There is already a visitor in PlanPrinter that can take care of the NestedLoopPhase, didn't see that before.